src/Types/accounting.ts

```diff
@@ -7,4 +7,9 @@ export type FlatOrder = {
 }
 
 // position is false if long
-export type FlatOrderWithSide = FlatOrder & { position: boolean }
\ No newline at end of file
+export type FlatOrderWithSide = FlatOrder & { position: boolean }
+
+export type Position = {
```

Comment:
Might be some naming confusion with this since I previously called position a boolean. Its called position in contracts so maybe we should think of a wording change for the other type of position.

Reply:
updated `boolean` to instead be called `isSell`

Reply:
that renaming looks like it has the potential to mess a bunch of stuff up across different repo's. I might rename the `Position` type to `Balance`